From 361cec592a9dc48f115c2bde598fd464bb203fe0 Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Sat, 31 Jan 2015 21:24:12 +0100
Subject: [PATCH 130/133] text: Change default text splash's colors

As per GNOME mockups:
https://wiki.gnome.org/Design/OS/Boot

Note that this also removes the blue terminal color setting, as it is
not used in the text plugin.

https://bugs.freedesktop.org/show_bug.cgi?id=88888
---
 src/plugins/splash/text/plugin.c | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/src/plugins/splash/text/plugin.c b/src/plugins/splash/text/plugin.c
index 85cfda2..468bb89 100644
--- a/src/plugins/splash/text/plugin.c
+++ b/src/plugins/splash/text/plugin.c
@@ -171,14 +171,11 @@ view_start_animation (view_t *view)
 
         ply_terminal_set_color_hex_value (terminal,
                                           PLY_TERMINAL_COLOR_BLACK,
-                                          0x000000);
+                                          0x2e3436);
         ply_terminal_set_color_hex_value (terminal,
                                           PLY_TERMINAL_COLOR_WHITE,
                                           0xffffff);
         ply_terminal_set_color_hex_value (terminal,
-                                          PLY_TERMINAL_COLOR_BLUE,
-                                          0x3465a4);
-        ply_terminal_set_color_hex_value (terminal,
                                           PLY_TERMINAL_COLOR_BROWN,
                                           0x979a9b);
 
-- 
1.9.0

